pkg/pgmodel/ingestor/duplicates.go

```diff
@@ -0,0 +1,39 @@
+package ingestor
+
+import (
+	"sync"
+	"time"
+
+	"go.uber.org/atomic"
```

Comment:
blocking: why are we using uber's atomic instead of the standard golang one? We shouldn't introduce a new dependency like this without discussion